src/vs/workbench/browser/parts/editor/editorGroupsControl.ts

```diff
@@ -87,11 +90,18 @@ export interface IEditorGroupsControl {
 	dispose(): void;
 }
 
+interface CenteredLayoutData {
+	leftMarginRatio: number;
+	size: number;
+}
+
 /**
  * Helper class to manage multiple side by side editors for the editor part.
  */
 export class EditorGroupsControl extends Themable implements IEditorGroupsControl, IVerticalSashLayoutProvider, IHorizontalSashLayoutProvider {
 
+	private static readonly CENTERED_LAYOUT_DATA_STORAGE_KEY = 'centeredLayoutData';
```

Comment:
I would prefix the value of this key with 'workbench' to be aligned with similar storage keys

Reply:
done.